src/test/scala/com/microsoft/hyperspace/index/IndexLogManagerImplTest.scala

```diff
@@ -144,11 +144,27 @@ class IndexLogManagerImplTest extends SparkFunSuite with SparkInvolvedSuite with
   test("testUpdateLatestStableLog passes if latestStable.json can be created") {
     val path = new Path(testRoot, UUID.randomUUID().toString)
     val fs = path.getFileSystem(new Configuration)
-    FileUtils.createFile(fs, new Path(path, s"$HYPERSPACE_LOG/0"), "file contents")
+    FileUtils.createFile(
+      fs,
+      new Path(path, s"$HYPERSPACE_LOG/0"),
+      JsonUtils.toJson(getEntry("ACTIVE"))
+    )
```

Comment:
Can we put this in the previous line? (that's how auto-format is done)

Reply:
ok

Reply:
done

Reply:
I meant the parenthesis:
```
FileUtils.createFile(
  fs, new Path(path, s"$HYPERSPACE_LOG/0"), JsonUtils.toJson(getEntry("ACTIVE")))
```